action/config/remove.go

```diff
@@ -15,28 +15,38 @@
~)
~
```

Comment:
Going to skip this check because part of the reason why the function is so long is due to comments.

If we were to remove all comments, we'd be down to 50-60 LOC.